src/lib/data/data_types.ts

```diff
@@ -4,10 +4,11 @@ interface Comments {
   sga_treasurer_comment?: string;
 }
 // an Item will represent a single item in a budget
-interface Item extends Comments {
+interface Item extends Comments, Document {
+  budget_id: string;
   name: string;
   quantity: number;
-  unit_price: number;
+  cost: number;
```

Comment:
Question: is there a reason why you changed the name from `unit_price` to `cost`? If cost is meant to store the cost for all the items of that type (so price * quantity) then maybe something like `total_cost` might be more sutiable so nobody get's confused